clients/drcachesim/simulator/tlb.h

```diff
@@ -45,6 +45,10 @@ class tlb_t : public caching_device_t {
     void
     request(const memref_t &memref) override;
 
+    // TODO i#4816: This tag,pid pair lookup needs to be imposed on the parent
```

Comment:
What lookup are you referring to? is this a misplaced/stale comment?

Reply:
Will reword: the TLB simulator keys on the tag plus the pid, and when I went to have it disallow use of these hashtables (it inherits from the same cachine_device_t) I found it seems to not consistently use the tag,pid pair in some existing code; hence I filed #4816 and put some comments in several places.